nemo/collections/nlp/models/text2sparql/text2sparql_model.py

```diff
@@ -44,14 +44,6 @@ def input_types(self) -> Optional[Dict[str, NeuralType]]:
             "labels": NeuralType(('B', 'T'), ChannelType(), optional=True),
         }
 
-    @property
```

Comment:
Why is removing this necessary for 1.5.0?

Reply:
Couple of reasons

1) HF changed their return API to return 4 things rather than 3 like before.
2) The extra thing being return is a heavily listed tuple of tuple of tensors corresponding to keys and values for each layer I think. Which is complicated to specify neural types for.